images/bay-php/settings.php

```diff
@@ -250,9 +250,9 @@
 // Use mounted fileshare for temp directory unless explicity disabled with
 //  BAY_SHARED_TEMP_FILES=false
 if (getenv("BAY_SHARED_TEMP_FILES") !== "false") {
-  $config['system.file']['path']['temporary'] = sprintf("%s/tmp", $settings['file_private_path']);
+  $settings['file_temp_path'] = sprintf("%s/tmp", $settings['file_private_path']);
```

Comment:
Just for future us, the reference is https://drupal.stackexchange.com/a/233120. After Drupal 8.8 using `settings.php` changed format.